test/restDocs/test.spec.ts

```diff
@@ -45,30 +54,11 @@ const mySnapshotTest = async (name: string, action: () => Promise<void>) => {
 
   await action();
 
-  const colordiffBinPath = execSync('which colordiff || which diff', {
-    encoding: 'utf-8',
-  }).trim(); // recommended `sudo apt install colordiff`
-  const diffCommand = `${colordiffBinPath} ${snapshotFolder} ${resultFolder}`;
-
-  if (process.argv.includes('--update')) {
-    console.warn(`updating ${name} snapshot...`);
-    execSync(
-      `rm -rf ${snapshotFolder} && mkdir -p ${snapshotFolder} && cp -R ${resultFolder}/. ${snapshotFolder}`
+  readdirSync(resultFolder).forEach(file => {
+    expect(readFileSync(join(resultFolder, file))).toMatchSpecificSnapshot(
+      join(snapshotFolder, file + '.shot')
```

Comment:
Can we please use `.snap` extension, same as what Jest uses by default?

Reply:
Also I'd prefer to use template strings instead of concatenation
```ts
`${file}.snap`
```

Reply:
If using `snap` extension, jest will complain that the snapshot is obsolete.

Reply:
I have traced to [const testPath = snapshotResolver.resolveTestPath(snapshotFile);](https://github.com/facebook/jest/blob/main/packages/jest-snapshot/src/index.ts#L133-L147)
which indicates that the `results` folder should be replaced with `__tests__` and the `snapshots` folder with `__snaphosts__`. 

Reply:
But the compiled JavaScript behavior is different from the Typescript.

Reply:
We have to use [snapshotResolver](https://jestjs.io/docs/configuration#snapshotresolver-string) to fix this problem.